### ISSUE TITLE
Alt Attributes missing on images

### ISSUE THREAD

**Issue by aaronmauchley-adobe** [adobe/business-website#150]
Various images across the blog homepage and blog posts are missing alt attributes, this is an accessibility WCAG compliance concern.  It can be updated after deployment, I don’t think it is a no-go situation, just something that should be addressed and audited so someone can go in and add the attribute to images that require them (non-background images).

@davidnuescheler responded back to this issue mentioned in slack:  

i think this is general an authoring hygiene issue... as far as the alt texts on cards (hero images goes) the query-index.xlsx would give a comprehensive list of articles where the alt text is missing... at a quick glance ~437 article images are missing an alt text.

**Comment by stereoke** [comment]
@aaronmauchley-adobe tracking here and assigned to Porter: DOTCOM-54822
